clang/lib/Sema/SemaExpr.cpp

```diff
@@ -3453,6 +3453,10 @@ ExprResult Sema::BuildDeclarationNameExpr(const CXXScopeSpec &SS,
                                    NeedsADL, R.isOverloadedResult(),
                                    R.begin(), R.end());
 
+  if (ULE && R.isSingleResult() && R.getFoundDecl()->isInvalidDecl()) {
```

Comment:
```suggestion
  if (R.isSingleResult() && R.getFoundDecl()->isInvalidDecl()) {
```
AST Node `Create*` functions normally can't fail.

Reply:
Done, thanks